### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

File: examples/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart [hunk 1]
```diff
@@ -24,7 +24,7 @@ const List<Color> coolColors = const <Color>[
 const List<String> coolColorNames = const <String>[
   'Sarcoline', 'Coquelicot', 'Smaragdine', 'Mikado', 'Glaucous', 'Wenge',
   'Fulvous', 'Xanadu', 'Falu', 'Eburnean', 'Amaranth', 'Australien',
-  'Banan', 'Falu', 'Gingerline', 'Incarnadine', 'Labrabor', 'Nattier',
+  'Banan', 'Falu', 'Gingerline', 'Incarnadine', 'Labrador', 'Nattier',
```
Comment (jsoref) [inline examples/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart]: I'm not sure about this change

File: examples/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart [hunk 2]
```diff
@@ -678,7 +678,7 @@ List<Widget> buildTab2Conversation() {
         text: 'SJ',
         color: const Color(0xFF34CAD6),
       ),
-      text: "We'll send you our\nnewest Labrabor too!",
+      text: "We'll send you our\nnewest Labrador too!",
```
Comment (jsoref) [inline examples/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart]: I'm not sure about this change

File: examples/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart (hunk 1 above)
Comment (xster) [reply examples/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart]: The correction's right. I just took a bunch of random colors from http://mentalfloss.com/article/59328/19-colors-youve-probably-never-heard. But this is misspelled.

File: packages/flutter/lib/src/animation/tween.dart [hunk 3]
```diff
@@ -37,42 +37,42 @@ abstract class Animatable<T> {
 }
 
 class _AnimatedEvaluation<T> extends Animation<T> with AnimationWithParentMixin<double> {
-  _AnimatedEvaluation(this.parent, this._evaluatable);
+  _AnimatedEvaluation(this.parent, this._evaluable);
```
Comment (xster) [inline packages/flutter/lib/src/animation/tween.dart]: Based on this graph, I think it's ok 
https://english.stackexchange.com/a/312764

File: packages/flutter/lib/src/rendering/sliver.dart [hunk 4]
```diff
@@ -71,7 +71,7 @@ AxisDirection applyGrowthDirectionToAxisDirection(AxisDirection axisDirection, G
 /// This function is useful in [RenderSliver] subclasses that are given both an
 /// [ScrollDirection] and a [GrowthDirection] and wish to compute the
 /// [ScrollDirection] in which growth will occur.
-ScrollDirection applyGrowthDirecitonToScrollDirection(ScrollDirection scrollDirection, GrowthDirection growthDirection) {
+ScrollDirection applyGrowthDirectionToScrollDirection(ScrollDirection scrollDirection, GrowthDirection growthDirection) {
```
Comment (xster) [inline packages/flutter/lib/src/rendering/sliver.dart]: This is unfortunate. It'll be API breaking but I think it's deep enough that it's hopefully not really used. I didn't find any references to it in Github at least.

File: packages/flutter/lib/src/widgets/framework.dart [hunk 5]
```diff
@@ -4292,7 +4292,7 @@ abstract class RenderObjectElement extends Element {
   /// and then returns the new child list.
   ///
   /// During this function the `oldChildren` list must not be modified. If the
-  /// caller wishes to remove elements from `oldChildren` re-entrantly while
+  /// caller wishes to remove elements from `oldChildren` reentrantly while
```
Comment (xster) [inline packages/flutter/lib/src/widgets/framework.dart]: I'd just leave this as is.

File: packages/flutter/test/gestures/pointer_router_test.dart [hunk 6]
```diff
@@ -77,7 +77,7 @@ void main() {
     expect(callbackRan, isFalse);
   });
 
-  test('Per-pointer callbacks cannot re-entrantly add global routes', () {
+  test('Per-pointer callbacks cannot reentrantly add global routes', () {
```
Comment (xster) [inline packages/flutter/test/gestures/pointer_router_test.dart]: ditto

File: packages/flutter/test/widgets/form_test.dart [hunk 7]
```diff
@@ -33,7 +33,7 @@ void main() {
     Future<Null> checkText(String testValue) async {
       await tester.enterText(find.byType(TextFormField), testValue);
       formKey.currentState.save();
-      // pump'ing is unnecessary because callback happens regardless of frames
+      // pumping is unnecessary because callback happens regardless of frames
```
Comment (xster) [inline packages/flutter/test/widgets/form_test.dart]: I'd leave these as well

File: packages/flutter_tools/lib/src/commands/upgrade.dart [hunk 8]
```diff
@@ -53,7 +53,7 @@ class UpgradeCommand extends FlutterCommand {
       throwToolExit(null, exitCode: code);
 
     // Check for and download any engine and pkg/ updates.
-    // We run the 'flutter' shell script re-entrantly here
+    // We run the 'flutter' shell script reentrantly here
```
Comment (xster) [inline packages/flutter_tools/lib/src/commands/upgrade.dart]: ditto

**Review by xster — APPROVED** [review]
Thanks for your contribution. 

Looks good. Unfortunately there is one API breaking change in there but it's deep enough hopefully it's not used. 

Just a few minor comments then I'll merge it for you.

**Comment by jsoref** [comment]
Oh, I should have flagged that there were API breaks, I often do. But yes, I was kinda hoping you'd be ok w/ them. (It's, of course, possible to add shims if necessary,)

@xster : I can't tell if you want me to push an update (I can do that), or if you'll just do the bits yourself.
I can see that AppVeyor is still building.

**Comment by mants22** [comment]
Just can you push the update please

Mr Mantas M.
________________________________

**Comment by jsoref** [comment]
@xster , @mants22 : updated (and all checks pass)

**Comment by xster** [comment]
Thanks!

**Comment by Hixie** [comment]
timeless? is that you?

**Pull request merged**